### ISSUE TITLE
language setting

### ISSUE THREAD

**Issue by ppete2** [waymarkedtrails/waymarked-trails-site#348]
Please take a look at https://cycling.waymarkedtrails.org/#routelist?map=18!48.3113!14.2514

in waymarkedtrail's setting i've choosen: language = Browsersettings. In the language settings of my browser (Firefox) I've activated Deutsch [de].

In the routelist the continental routes are written in their German version - correct behaviour, e.g. https://www.openstreetmap.org/relation/2938 is written "EuroVelo 6 - Atlantik-Schwarzes Meer"

But not the national route https://www.openstreetmap.org/relation/23400 which is written with its English name "Danube Cycle Path (part Upper Austria)" although the name-tag consists of a German name "Donauweg (Oberösterreich)"

An interesting fact is, that if switching waymarkedtrail's setting to: language = German, now also the national route is written in its German name.

Is this a bug or intentional, that the national route - as opposed to continental routes - is not rendered in German if language setting = "Browsersettings"

**Comment by lonvia** [comment]
The problem here is that there is no name:de tag on that route and waymarkedtrails does not make any guesses as to what language the name tag may be in. In your browser you had likely set 'de,en' as language. So waymarkedtrails looks for 'name:de,name:en,name' in that order and uses the first it finds. When you set a fixed language 'Deutsch' in the settings, then there is no fallback to English anymore. waymarkedtrails just looks for 'name:de,name'.

Long story short: the behaviour is correct and intended. If a route has localised name tags, it should always repeat the 'name' tag in the localised 'name:<lang>' tag.
